server/bin/pbench-backup-tarballs.py

```diff
@@ -359,6 +365,14 @@ def backup_data(lb_obj, s3_obj, config, logger):
         resultname = tar.name
         controller_path = tar.parent
         controller = controller_path.name
+        try:
+            # This tool can't see a dataset until it's been prepared either
+            # by server PUT or by pbench-server-prep-shim-002.py; in either
+            # case, the Dataset object must already exist.
+            dataset = Dataset.attach(controller=controller, path=resultname)
+        except DatasetError as e:
+            logger.info("Trouble tracking {}:{}: {}", controller, resultname, str(e))
+            dataset = None
```

Comment:
Seems like this should at least be a `warning`.

Should we consider logging an error here and not doing the backup because the assumptions are not correct?

Reply:
Do we ever want to _not_ do a backup? I thought not...